pkg/vpcmodel/multiExplainability.go

```diff
@@ -169,6 +169,9 @@ func collectNodesForExplanation(cConfigs *MultipleVPCConfigs, conns map[string]*
 			}
 		}
 	}
+	if cConfigs.publicNetworkNode != nil {
```

Comment:
What is the situation in which it is legit  cConfigs.publicNetworkNode will be nil?
Shouldn't this node be always present?